test/functional/wallet_abandonconflict.py

```diff
@@ -163,6 +164,34 @@ def run_test(self):
         self.connect_nodes(0, 1)
         self.sync_blocks()
 
+        tx_list = self.nodes[0].listtransactions()
+
+        conflicted = [tx for tx in tx_list if tx["confirmations"] < 0]
+        assert_equal(4, len(conflicted))
+
+        wallet_conflicts = [tx for tx in conflicted if tx["walletconflicts"]]
```

Comment:
Probably it would read a little clearly if  instead of counting `walletconflicts` in `conflicted`, we count it in `tx_list`?

That will give 3 wallet conflicts. Not that its important for assertion, because the third conflict is just asserted below this. But it did confused me for a moment in first pass why only 2 `walletconflicts`instead of 3.

Reply:
That would mean iterating through the (quite a bit longer) `tx_list`, and I only want to look through the conflicted transactions here to check them versus the double spend. Happy to look at a code proposal though.

Reply:
Ya I think that makes sense, we do need to iterate over a longer list then. Its not that important. It was a dumb confusion I had while reading. But the code is correct.